profiles/dini.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
+// In case everything else fails, we want the error
+window.addEventListener("error", ev => {
+  console.error(ev.error, ev.message, ev);
+});
+
+const modules = [
+  // order is significant
+  import("../src/core/base-runner.js"),
+  import("../src/core/ui.js"),
+  import("../src/core/location-hash.js"),
+  import("../src/core/l10n.js"),
+  import("../src/dini/defaults.js"),
+  import("../src/core/style.js"),
+  import("../src/w3c/style.js"),
+  import("../src/core/data-include.js"),
+  import("../src/core/markdown.js"),
+  import("../src/core/reindent.js"),
+  import("../src/dini/headers.js"),
+  import("../src/w3c/abstract.js"),
+  import("../src/core/data-transform.js"),
+  import("../src/core/data-abbr.js"),
+  import("../src/core/inlines.js"),
+  import("../src/w3c/conformance.js"),
+  import("../src/core/dfn.js"),
+  import("../src/core/pluralize.js"),
+  import("../src/core/examples.js"),
+  import("../src/core/issues-notes.js"),
+  import("../src/core/best-practices.js"),
+  import("../src/core/figures.js"),
+  import("../src/core/webidl.js"),
+  import("../src/core/data-cite.js"),
+  import("../src/core/biblio.js"),
+  import("../src/core/webidl-index.js"),
+  import("../src/core/link-to-dfn.js"),
+  import("../src/core/render-biblio.js"),
+  import("../src/core/contrib.js"),
+  import("../src/core/fix-headers.js"),
+  import("../src/core/structure.js"),
+  import("../src/core/informative.js"),
+  import("../src/core/id-headers.js"),
+  import("../src/core/caniuse.js"),
+  import("../src/core/mdn-annotation.js"),
+  import("../src/ui/save-html.js"),
+  import("../src/ui/dfn-list.js"),
+  import("../src/ui/about-respec.js"),
+  import("../src/core/seo.js"),
+  import("../src/core/highlight.js"),
+  import("../src/core/webidl-clipboard.js"),
+  import("../src/core/data-tests.js"),
```

Comment:
This is mostly for Web Platform Tests... so might not be useful. 

```suggestion
```

Reply:
Fixed with https://github.com/w3c/respec/pull/2739/commits/7c31731400bb30ae1a34490d53aa8e1ae342d546